App/views.py

```diff
@@ -1,15 +1,22 @@
 from django.http import HttpResponse
 from django.shortcuts import render
+import math
 import json
 import requests
 import toolforge
 
 def main2(request):
     if(request.method=='POST'):
+        flag = 1
         usr = request.POST['username']
+        a = 1 if re.match("^[a-zA-Z0-9_ ]*$", s) else 0
+        if(!a):
+            flag=0
```

Comment:
Not much point in setting this as you are exiting the function anyway.

Reply:
Done.